decidim-admin/app/packs/stylesheets/decidim/admin/_moderations.scss

```diff
@@ -29,6 +29,14 @@
     list-style: none;
     margin: 0;
 
+    li a {
+      @apply flex items-center;
+
+      svg {
+        @apply mx-1;
```

Comment:
```suggestion
        @apply ml-1;
```
Lets change this to only utilise the left margin element only, to make both svg's equal in terms of space.

<img width="273" alt="Screenshot 2024-11-29 at 08 18 20" src="https://github.com/user-attachments/assets/8a1f4ca0-c6dc-4ffc-9f1e-b25f4ad2f6a1">

Reply:
@greenwoodt The screenshots are visible in the PR description.